08LessonEight/homework/css/exercise-06-style.css

```diff
@@ -1 +1,36 @@
-/*Add CSS rules*/
\ No newline at end of file
+* {box-sizing: border-box;}
+body {
+  font-family: helvetica;
+  color: #555555;
+  max-width: 800px;
+  margin: 0 auto;
+}
+.parent {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 20px;
+}
+section {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+article {
+  margin-right: 25px;
+}
```

Comment:
Your entire middle section is slightly off-centered because of the margin on the far-right article. To remedy, you can target it with `article:last-child { margin-right:0;}`